nightmare/paths/metadata.spec.js

```diff
@@ -11,213 +13,395 @@ describe('metadata path', () => {
+        multidaterangeFromInputTwo:
+          '#metadataForm > div:nth-child(3) > div:nth-child(9) > ul > li.wide > div > div:nth-child(2) > div > div.DatePicker__From > div > input',
+        multidaterangeToInputTwo:
+          '#metadataForm > div:nth-child(3) > div:nth-child(9) > ul > li.wide > div > div:nth-child(2) > div > div.DatePicker__To > div > input',
+        multidaterangeRemoveDateOne:
+          '#metadataForm > div:nth-child(3) > div:nth-child(9) > ul > li.wide > div > div:nth-child(1) > div > button',
+        multidaterangeRemoveDateTwo:
+          '#metadataForm > div:nth-child(3) > div:nth-child(9) > ul > li.wide > div > div:nth-child(2) > div > button',
+        richText:
+          '#metadataForm > div:nth-child(3) > div:nth-child(10) > ul > li.wide > div > div.tab-content.tab-content-visible > textarea',
+        linkLabel:
+          '#metadataForm > div:nth-child(3) > div:nth-child(11) > ul > li.wide > div > div > div:nth-child(1) > input',
+        linkUrl:
+          '#metadataForm > div:nth-child(3) > div:nth-child(11) > ul > li.wide > div > div > div:nth-child(2) > input',
+        image:
+          '#metadataForm > div:nth-child(3) > div:nth-child(12) > ul > li.wide > div > div > textarea',
+        media:
+          '#metadataForm > div:nth-child(3) > div:nth-child(14) > ul > li.wide > div > div > textarea',
+        geolocationLat:
+          '#metadataForm > div:nth-child(3) > div:nth-child(15) > ul > li.wide > div > div.form-row > div:nth-child(1) > input',
+        geolocationLon:
+          '#metadataForm > div:nth-child(3) > div:nth-child(15) > ul > li.wide > div > div.form-row > div:nth-child(2) > input',
+        save:
+          '#app > div.content > div > div > aside.side-panel.metadata-sidepanel.is-active > div.sidepanel-footer > span > button.btn.btn-success',
+      },
+    };
+
+    it('should test all the properties', async () => {
+      await nightmare
+        .waitToClick(selectors.settingsView.templatesButton)
+        .waitToClick(selectors.settingsView.addNewTemplate);
+
+      const numberOfProps = await nightmare.evaluate(
+        selector => document.querySelectorAll(selector).length,
+        localSelectors.templateProperties
+      );
+
+      if (numberOfProps !== 15) {
+        fail('amount of props have change, update this test');
+      }
+    });
+
+    it('should create a template with all the properties', async () => {
+      await nightmare
+        .waitToClick(selectors.settingsView.templatesButton)
+        .waitToClick(selectors.settingsView.addNewTemplate)
+        .write(selectors.settingsView.documentTemplateNameForm, 'All props');
+
+      for (let propIndex = 1; propIndex <= 15; propIndex += 1) {
+        await nightmare.waitToClick(localSelectors.propertiesButtons(propIndex));
+      }
+
+      await nightmare
+        .waitToClick(localSelectors.editRelationshipProperty)
+        .select(localSelectors.relationshipPropertyType, '5a8480eac464318833d9b54e')
```

Comment:
can you use the selectByLabel action you developed on this same pr for all selects ? i thin will help on readability.